### PR TITLE
Add dapp api helper - Closes #290

### LINKED ISSUE #290: Add missing dapp API functions
To access the following endpoints from old Lisk Core API:

- GET /api/dapps - GetDapps
- GET /api/dapps/get - GetDapp
- GET /api/dapps/categories - GetCategories

### PR DESCRIPTION
**Pull request by Tosch110**
Closes #290

### CODE REVIEW

File: src/api/liskApi.js [hunk 1]
```diff
@@ -372,6 +372,40 @@ LiskAPI.prototype.getVoters = privateApi.wrapSendRequest(GET, 'voters', username
...
+/**
+ * @method getDappCategory
+ * @param category
+ * @param optionsOrCallback
+ * @param callbackIfOptions
+ *
+ * @return API object
+ */
+
+LiskAPI.prototype.getDappCategory = privateApi.wrapSendRequest(GET, 'dapps', category => ({ category }));
```
Comment (willclarktech) [inline src/api/liskApi.js]: `getDappsByCategory`?

File: test/api/liskApi.js [hunk 2]
```diff
@@ -399,6 +399,44 @@ describe('Lisk.api()', () => {
...
+				LSK.getDapp(transactionId, callback);
+				(LSK.sendRequest.calledWithExactly(GET, 'dapps', options, callback)).should.be.true();
+			});
+		});
+
+		describe('#getDapps', () => {
+			it('should get dapps with options', () => {
+				const options = {
+					limit: defaultRequestLimit,
+					offset: defaultRequestOffset,
```
Comment (willclarktech) [inline test/api/liskApi.js]: These options don't apply to this endpoint.

File: test/api/liskApi.js [hunk 3]
```diff
@@ -399,6 +399,44 @@ describe('Lisk.api()', () => {
...
+				LSK.getDapps(options, callback);
+				(LSK.sendRequest.calledWithExactly(GET, 'dapps', options, callback)).should.be.true();
+			});
+		});
+
+		describe('#getDappCategory', () => {
+			it('should get getDappCategory', () => {
+				const options = {
+					limit: defaultRequestLimit,
+					offset: defaultRequestOffset,
```
Comment (willclarktech) [inline test/api/liskApi.js]: As above.

File: test/api/liskApi.js (hunk 2 above)
Comment (Tosch110) [inline test/api/liskApi.js]: They should get added soon: 
https://github.com/LiskHQ/lisk/issues/240#issuecomment-323074720

**Pull request merged**